test/sanitizer_suppressions/tsan

```diff
@@ -39,5 +30,6 @@ race:libzmq
 # https://github.com/bitcoin/bitcoin/issues/20618
 race:CZMQAbstractPublishNotifier::SendZmqMessage
 
-# https://github.com/bitcoin/bitcoin/pull/20218, https://github.com/bitcoin/bitcoin/pull/20745
-race:epoll_ctl
+# Uninstrumented external libraries.
```

Comment:
Would it help to build depends with instrumentation?

Reply:
Good question. Did not try it yet.

Reply:
I think eventually we don't want libraries that are broken, but we can look into this after bdb is removed